### PR TITLE
fix #275539: use correct paths for retrieving Qt installation files

### PR DESCRIPTION
**Pull request by dmitrio95**
See https://musescore.org/ru/node/275539 and https://doc.qt.io/qt-5/qmake-environment-reference.html.

### CODE REVIEW

File: mscore/CMakeLists.txt [hunk 1]
```diff
@@ -612,8 +612,8 @@ if (MINGW)
       ${MINGW_ROOT}/lib/libvorbis.dll
       ${MINGW_ROOT}/lib/libvorbisfile.dll
       ${MINGW_ROOT}/opt/bin/ssleay32.dll
-      ${QT_INSTALL_PREFIX}/bin/libEGL.dll
-      ${QT_INSTALL_PREFIX}/bin/libGLESv2.dll
+      ${QT_INSTALL_BINS}/bin/libEGL.dll
+      ${QT_INSTALL_BINS}/bin/libGLESv2.dll
```
Comment (Jojo-Schmitz) [inline mscore/CMakeLists.txt]: If `${QT_INSTALL_LIBS} includes `/lib` I'd expect `${QT_INSTALL_BINS}` to include `/bin`. Wrong?

File: mscore/CMakeLists.txt [hunk 2]
```diff
@@ -840,17 +840,17 @@ else (MINGW)
...
-                  "${QT_INSTALL_PREFIX}/bin/Qt5QuickWidgets.dll" "${QT_INSTALL_PREFIX}/bin/Qt5Positioning.dll"
-                  "${QT_INSTALL_PREFIX}/bin/libEGL.dll" "${QT_INSTALL_PREFIX}/bin/libGLESv2.dll"
+                  "${QT_INSTALL_BINS}/Qt5Core.dll"  "${QT_INSTALL_BINS}/Qt5Gui.dll"  "${QT_INSTALL_BINS}/Qt5Help.dll"
+                  "${QT_INSTALL_BINS}/Qt5Network.dll"  "${QT_INSTALL_BINS}/Qt5PrintSupport.dll"
+                  "${QT_INSTALL_BINS}/Qt5Qml.dll"  "${QT_INSTALL_BINS}/Qt5Quick.dll"  "${QT_INSTALL_BINS}/Qt5Sql.dll"
+                  "${QT_INSTALL_BINS}/Qt5Svg.dll"  "${QT_INSTALL_BINS}/Qt5Widgets.dll"  "${QT_INSTALL_BINS}/Qt5Xml.dll"
+                  "${QT_INSTALL_BINS}/Qt5XmlPatterns.dll"
+                  "${QT_INSTALL_BINS}/Qt5WebChannel.dll"
+                  "${QT_INSTALL_BINS}/Qt5QuickWidgets.dll" "${QT_INSTALL_BINS}/Qt5Positioning.dll"
+                  "${QT_INSTALL_BINS}/bin/libEGL.dll" "${QT_INSTALL_BINS}/bin/libGLESv2.dll"
```
Comment (Jojo-Schmitz) [inline mscore/CMakeLists.txt]: same here

File: mscore/CMakeLists.txt (hunk 1 above)
Comment (dmitrio95) [reply mscore/CMakeLists.txt]: Yes, looks like a mistake, thank you.

**Comment by Jojo-Schmitz** [comment]
I guess parts of b4befe26 are needed here too

**Comment by dmitrio95** [comment]
Updated paths for b4befe26c2c7368bafce6beecf37650fda6bb7cb.

**Pull request merged**